### PR TITLE
Update root readme: Add little quickstart blurb

### PR DESCRIPTION
**Pull request by aschokking**
It took me longer than it should have to get the ui + api working locally at first. This was because I got lost in the different READMEs and what the ui server was or wasn't doing. This might help the next person get rolling a little faster 🤞

### CODE REVIEW

**Comment by andrewhead** [comment]
Lgtm! Maybe direct readers to the `ui/README.md` as well, which has additional context about what to expect to see when the reader app launches and building the app?

File: README.md [hunk 1]
```diff
@@ -16,6 +16,23 @@ papers and bounding boxes for those entities.
...
+In order to run the reader app locally (pointing to the production api) you only need to do 
+
+```bash
+cd ui
+npm install 
+npm start
+```
+
+However, if you want to run the api locally as well you'll have to:
+- Do the 1 time setup instructions for the api detailed in [api/README.md#one-time-setup](api/README.md#one-time-setup)
```
Comment (andrewhead) [inline README.md:30]: Readers should also do the `npm install` in the API directory before following the instructions in one-time setup

**Review by andrewhead — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aschokking) [reply README.md:30]: yeah that's fair, I was trying to balance how much detail to put in here but you're right that one is worth including

Comment (aschokking) [reply README.md]: Changed this up so the api/README.md covered the one time setup in the right spot and just referenced those instructions as a whole

**Pull request merged**